xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-send/xwiki-platform-mail-send-default/src/main/java/org/xwiki/mail/script/ScriptMimeMessage.java

```diff
@@ -129,6 +129,28 @@ public BodyPart addPart(String mimeType, Object content, Map<String, Object> par
         return bodyPart;
     }
 
+    /**
+     * Add some content to the mail to be sent. Can be called several times to add different content type to the mail.
+     *
+     * @return the Mime Body Part object that was added. Returning it allows script to make modifications to that body
+     *         part after it's been set (get/set some headers, etc)
+     * @param bodyPart the Mime Body Part object to be added
+     */
+    public BodyPart addPart(BodyPart bodyPart)
+    {
+        BodyPart temp = bodyPart;
```

Comment:
I prefer naming variables by what they represent. In this case for example: `returnedBodyPart` or `result`.